mjpeg.go

```diff
@@ -99,6 +100,7 @@ func (s *Stream) Update(b []byte) error {
 		select {
 		case c <- b:
 		default:
+			fmt.Println("no updates")
```

Comment:
Do not call fmt.Println here

Reply:
Sorry, I am testing now) It not fixed my bugs, fmt for debug)